package-lock.json

```diff
@@ -18126,6 +18156,17 @@
         "events": "^2.0.0"
       },
       "dependencies": {
+        "@cardano-foundation/ledgerjs-hw-app-cardano": {
```

Comment:
`trezor-connect` depends on `lederjs-hw-app-cardano`? which btw is 1.0.6 and not 1.0.7 🤔

Reply:
yeah that's really strange. Not sure what would cause this

Reply:
@nicarq @SebastienGllmt 
![image](https://user-images.githubusercontent.com/19986226/61926226-adff4680-afaa-11e9-9741-c4ece6b1a873.png)

Probably its Git bug, in IDE it looks perfect.
there is ~1500 line difference in those two bocks.